bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -15,8 +15,14 @@
 
 		<channels>
 			<channel id="color" typeId="system.color"/>
+			<channel id="color_temperature" typeId="system.color-temperature"/>
+			<channel id="color_temperature_abs" typeId="system.color-temperature-abs"/>
```

Comment:
It would be cool if you could respect the conventions that were recently clarified:
https://next.openhab.org/docs/developer/guidelines.html#naming-convention
What is expected as channel names in your case is `color-temperature `and `color-temperature-abs`.

Reply:
I would recommend sticking to the existing convention in the binding for consistency.